common/common/src/main/kotlin/com/cosmotech/api/config/CsmOpenAPIConfiguration.kt

```diff
@@ -69,10 +72,19 @@ internal class CsmOpenAPIConfiguration(val csmPlatformProperties: CsmPlatformPro
     // so as to have the base URL auto-generated based on the incoming requests
     openAPI.servers = listOf()
 
-    val oauth2Scopes = openAPI.components.securitySchemes["oAuth2AuthCode"]?.flows?.implicit?.scopes
-    if (csmPlatformProperties.azure != null) {
-      oauth2Scopes?.clear()
-      oauth2Scopes?.put("${csmPlatformProperties.azure!!.appIdUri}/platform", "Platform scope")
+    if (csmPlatformProperties.identityProvider != null) {
```

Comment:
Seems not retro-compatible (no csmPlatformProperties.identityProvider defined).
Handle the null case with the delete block of csmPlatformProperties.azure != null?